### DISCUSSION TITLE
Align easepick popup(modal) on the right end of input

### DISCUSSION THREAD

**Discussion by fbenavidesd** [easepick/easepick#236] [Q&A]
I use easepick with grid: 2, calendars: 2. Is there some option to make the popup(modal) is shown align to the right end of input. 
thanks you

![Captura de pantalla 2024-02-08 100018](https://github.com/easepick/easepick/assets/8224297/334c22b4-82b9-4418-9013-806370c0b20d)

**Comment by cavasinf** [comment]
At the moment no.. Waiting for https://github.com/easepick/easepick/pull/216

You can:
1. Tweak it manually with position `top`/`left` (`easepick-wrapper` class is absolute)
2. Use grid: 1, calendars: 2 instead.

**Reply by fbenavidesd** [reply]
I try with your first idea but couldn´t do it. 
Thanks any way. Great component
